app/controllers/register/establishers/company/director/CheckYourAnswersController.scala

```diff
@@ -141,22 +141,15 @@ class CheckYourAnswersController @Inject()(appConfig: FrontendAppConfig,
       else
         Seq(companyDirectorDetails, companyDirectorContactDetails)
 
-      Future.successful(Ok(check_your_answers(
+      Future.successful(Ok(checkYourAnswers(
         appConfig,
         answerSections,
```

Comment:
can we use the SchemeTaskList onPageLoad here just for consistency?

Reply:
Done, removed navigator as not needed anymore